src/gui/qgsmaptoolidentify.cpp

```diff
@@ -1054,6 +1054,18 @@ bool QgsMapToolIdentify::identifyRasterLayer( QList<IdentifyResult> *results, Qg
   derivedAttributes.insert( derivedAttributesForPoint( QgsPoint( pointInCanvasCrs ) ) );
 #endif
 
+  // Get raster row and column of clicked point
+  // Thanks to the Value Tool plugin developers for the math
+  const QgsRectangle extent = dprovider->extent();
+  const double xres = extent.width() / dprovider->xSize();
+  const double yres = extent.height() / dprovider->ySize();
+
+  const int raster_row = std::floor( ( extent.yMaximum() - point.y() ) / yres );
```

Comment:
```suggestion
  const int rasterRow = std::floor( ( extent.yMaximum() - point.y() ) / yres );
```

Qgis code convention is camel case for variable names

Reply:
My Python brain was active, thanks.